CodeStarterCamp_Week3/week3_step1.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  week3_step1.swift
+//  CodeStarterCamp_Week3
+//
+//  Created by jjudy on 2023/09/11.
+//
+
+import Foundation
+
+enum Coffee: String {
+    case americano = "americano"
+    case latte = "latte"
+    case macchiato = "macchiato"
+    case flatWhite = "flatWhite"
+    case frappuccino = "frappucino"
+}
+
+class Person {
+    
+    enum Gender {
+        case male
+        case female
+    }
+    
+    var name: String
+    var age: Int
+    var gender: Gender
+    var money: Int
+    lazy var nickName: String = String(self.name.dropFirst(1))
+ 
+    // 이니셜라이저
+    init(name: String, age: Int, gender: Gender, money: Int) {
+        self.name = name
+        self.age = age
+        self.gender = gender
+        self.money = money
+    }
```

Comment:
```suggestion
    init(name: String, age: Int, gender: Gender, money: Int) {
        self.name = name
        self.age = age
        self.gender = gender
        self.money = money        
        self.nickName = name.dropFirst()
    }
```
이렇게 생성자 파라미터로 받은 name을 이용해 nickName을 초기화하면 될 것 같아요.
init() 메소드 안에서 이렇게 가공한 후 초기화해줘도 상관없습니다.
이러면 lazy 키워드가 필요없겠죠?